.github/workflows/ci.yml

```diff
@@ -29,6 +29,21 @@ jobs:
       matrix:
         NXF_VER:
           - "24.10.0"
+          - "latest-everything"
+        profile:
+          - "conda"
+          - "docker"
+          - "singularity"
+        test_name:
+          - "test"
+        isMaster:
```

Comment:
Is there a specific reason not to use `snake_case` here?

Reply:
This is taken from the other nf-core repos and I followed the same.

Reply:
I think we should follow the proper yaml/Github Actions format here, as it was probably accidentally introduced in nf-core (probably because this casing is used in the Java/Groovy world a bit).